jaxrs-api/src/main/java/jakarta/ws/rs/core/NewCookie.java

```diff
@@ -423,14 +424,8 @@ public String toString() {
      */
     @Override
     public int hashCode() {
-        int hash = super.hashCode();
-        hash = 59 * hash + (this.comment != null ? this.comment.hashCode() : 0);
-        hash = 59 * hash + this.maxAge;
-        hash = 59 + hash + (this.expiry != null ? this.expiry.hashCode() : 0);
-        hash = 59 * hash + (this.secure ? 1 : 0);
-        hash = 59 * hash + (this.httpOnly ? 1 : 0);
-        hash = 59 * hash + (this.sameSite != null ? this.sameSite.ordinal() : 0);
-        return hash;
+        return Objects.hash(this.getName(), this.getValue(), this.getVersion(), this.getPath(), this.getDomain(),
```

Comment:
As there's no other object in scope here, my preference would be to omit the explicit qualification with "this.".

Shorter is better, less code is better code, etc.

Reply:
I respect your personal style preference, but I do not share it, and the original code already used "this". So feel free to open a PR for removal of "this" whereever you like it, but let's concentrate in *this* PR on the its own topic.

Reply:
I would drop `this` as well. May have been there before if method was auto-generated, but otherwise, we should only need that in constructors really.

Reply:
Sigh. I hoped we could agree to separate our personal likes and dislikes from the general benefit of a PR ("SoC"). Anyways, removed "this" in https://github.com/eclipse-ee4j/jaxrs-api/pull/1075/commits/666d7513ff6a251f7263bb03ddab0c84791257c5.